int64 skill_type
string skill_description
int64 meta_skill_type
int64 meta_skill_id

# Sensor topic to subscribe to
string[] sensor_topics
uint64[] sensor_value_sizes
float64[] initial_sensor_values

# traj gen
int64 traj_gen_type
uint64 num_traj_gen_params
float64[] traj_gen_params

# fbc
int64 feedback_controller_type
uint64 num_feedback_controller_params
float64[] feedback_controller_params

# termination
int64 termination_type
uint64 num_termination_params
float64[] termination_params 

# timer
int64 timer_type
uint64 num_timer_params
float64[] timer_params
---
# Result - happens when the skill terminates
uint64 num_execution_result
float64[] execution_result
---
# Feedback message - happens during skill execution 
uint64 num_execution_feedback
float64[] execution_feedback
